cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/aggregation/WindowAggregationDefinition.java

```diff
@@ -211,7 +306,7 @@ public Builder windowFrameType(WindowFrameType windowFrameType) {
      * Any existing order by expression list is overwritten.
      * @return a {@link Builder} with the currently built {@link WindowAggregationDefinition}.
      */
-    public Builder orderBy(Expression expression, OrderBy orderBy) {
+    public Builder orderBy(@Nullable Expression expression,  @Nullable OrderBy orderBy) {
```

Comment:
These shouldn't be nullable. The order by expression and order parameter are required when defining the ordering criteria.

Also, if the list is null, maybe you should initialize the list before adding an expression

Reply:
In the existing windowsaggregation plugin, order by field is accepting even if one of the fields(column name or order by asc or desc) is entered,Based on that added @nullable.Will raise jira to fix the existing behaviour.Removed nullable from this method